packages/legend-shared/src/format/__tests__/FormatterUtils.test.ts

```diff
@@ -91,6 +79,33 @@ test(unitTest('Camel/Pascal case check'), () => {
   expect(isCamelCase('AAasd')).toBe(false);
 });
 
+test(unitTest('Prettify CONST name with capitaliztions'), () => {
+  expect(prettyCONSTName('fiveTwoEight')).toEqual('Five Two Eight');
+  expect(prettyCONSTName('FIVETwoEight')).toEqual('FIVE Two Eight');
+  expect(prettyCONSTName('fiveTWOEight')).toEqual('Five TWO Eight');
+  expect(prettyCONSTName('fiveTwoEIGHT')).toEqual('Five Two EIGHT');
+  expect(prettyCONSTName('   fiveTwoEIGHT   ')).toEqual('Five Two EIGHT');
+  expect(prettyCONSTName('five_Two_EIGHT')).toEqual('Five Two Eight');
+  expect(prettyCONSTName('five5TWOEight')).toEqual('Five 5TWO Eight');
+  expect(prettyCONSTName('five5TwoEIGHT')).toEqual('Five 5 Two EIGHT');
+  expect(prettyCONSTName('five5TWOEight9Two')).toEqual('Five 5TWO Eight 9 Two');
```

Comment:
Aren't we separating integers here? Shouldn't it look like `Five 5 TWO Eight 9 Two`?

Reply:
Yes true, should have spaces between those as well 👍